src/message/MessageContainer.js

```diff
@@ -16,29 +18,60 @@ const htmlToDomTree = html => {
   return domTree;
 };
 
-export default class MessageContainer extends React.PureComponent {
+
+class MessageContainer extends React.PureComponent {
+  constructor() {
+    super();
+    this.state = {
+      showActionSheet: false
+    };
+  }
 
   handlePress = () => {
+    const isFromReply = false;
+    this.narrow(isFromReply);
+  };
+
+  narrow = (isFromReply: boolean) => {
     const { message, doNarrow } = this.props;
-    doNarrow(narrowFromMessage(message), message.id);
-  }
+    doNarrow(narrowFromMessage(message), message.id, isFromReply);
```

Comment:
Maybe just call doNarrow in place.
Some minimal duplication, when it is just a func call is ok, and even preferable.
In this case, the function name 'narrow' can be confused with 'doNarrow' and would at least require a better more descriptive name.